Filter/StringFilter.php

```diff
@@ -55,7 +55,10 @@ public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $data)
         if ($data['type'] == ChoiceType::TYPE_EQUAL) {
             $queryBuilder->setParameter($parameterName, $data['value']);
         } else {
-            $queryBuilder->setParameter($parameterName, sprintf($this->getOption('format'), $data['value']));
+            $queryBuilder->setParameter($parameterName, sprintf(
+                $this->getOption('format'),
+                addcslashes($data['value'], '%_')
```

Comment:
Hi, you are right, it is can used by someone like feature.
I think I can add new option which will be enable/disable escaping. 
Escaping strategy I will fix.